efficientdet/det_model_fn.py

```diff
@@ -615,6 +631,70 @@ def before_run(self, run_context):
         every_n_iter=params.get('iterations_per_loop', 100),
     )
     training_hooks.append(logging_hook)
+
+    if params["nvgpu_logging"]:
+      try:
+        from third_party.tools import nvgpu  # pylint: disable=g-import-not-at-top
+        from functools import reduce  # pylint: disable=g-import-not-at-top
+
+        def get_nested_value(d, path):
+          return reduce(dict.get, path, d)
```

Comment:
Can we move most of the code to nvgpu, so this file can be clean? thanks.

For example: nvgpu_gpu_info and commonsize and formatter_log can be moved to nvgpu.